Content.Shared/Nutrition/Satiation.cs

```diff
@@ -0,0 +1,97 @@
+using Content.Shared.Damage;
+using Robust.Shared.Serialization;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Generic;
+
+namespace Content.Shared.Nutrition.Components;
+
+[DataDefinition]
+[Serializable, NetSerializable]
+public sealed partial class Satiation
```

Comment:
Move all the static data here into a new SatiationPrototype, make two of em (one for thirst, one for hunger)

Reply:
All static data moved to new prototype SatiationPrototype